src/main/java/com/brsanthu/googleanalytics/GoogleAnalyticsConfig.java

```diff
@@ -30,13 +30,16 @@
  * @author Santhosh Kumar
  */
 public class GoogleAnalyticsConfig {
+
+    public static final int DEFAULT_MAX_HTTP_CONNECTIONS_PER_ROUTE = 10;
+
     private String threadNameFormat = "googleanalyticsjava-thread-{0}";
     private boolean enabled = true;
     private int minThreads = 0;
     private int maxThreads = 5;
     private int threadTimeoutSecs = 300;
     private int threadQueueSize = 1000;
-    private int maxHttpConnectionsPerRoute = 10;
+    private int maxHttpConnectionsPerRoute = DEFAULT_MAX_HTTP_CONNECTIONS_PER_ROUTE;
     private boolean useHttps = true;
```

Comment:
OkHttp doesn't support routing connection count configuration, and making this a constant allows me to check it in the OkHttp implementation so I can warn any library users if they attempt to use this config and OkHttp at the same time.